app/views/Import.js

```diff
@@ -41,25 +42,35 @@ const ImportScreen = props => {
       setImportResults(makeImportResults());
 
       const filePath = await pickFile();
-      if (filePath) {
-        const newLocations = await importTakeoutData(filePath);
-        if (newLocations.length) {
-          setImportResults(makeImportResults('label.import_success'));
-        } else {
-          setImportResults(makeImportResults('label.import_already_imported'));
-        }
+
+      const newLocations = await importTakeoutData(filePath);
+
+      if (newLocations.length) {
+        setImportResults(makeImportResults('import.success'));
```

Comment:
use full `t('import.success')` whenever you use i18n keys, or the i18n detection/extraction tool will not find the strings.